components/SearchBar/styles.ts

```diff
@@ -0,0 +1,49 @@
+import { CSSProperties } from 'react';
+
+export const searchBarPaddingStyles: CSSProperties = {
+  display: 'inline-flex',
```

Comment:
remember, we are using styled components not CSS properties. Please check Neha's recent styling sprint to see how styled components work.